src/framework/global/dataformatter.cpp

```diff
@@ -44,48 +44,25 @@ String DataFormatter::formatTimeSince(const Date& dateTime)
         return mtrc("global", "Today");
     }
 
-    if (days == 1) {
-        return mtrc("global", "Yesterday");
-    }
-
-    if (days < 7) {
-        return mtrc("global", "%1 days ago").arg(days);
+    if (days >= 1 && days < 7) {
+        return mtrc("global", "%n day(s) ago", nullptr, days);
```

Comment:
I'd prefer to keep "Yesterday", "Last week" and "Last month". Just because "Yesterday" sounds a lot better to me than "1 day ago".
(Apart from that, I think it's good to use %n day(s) etc. here, to account for languages that have different forms for "dual" and plural.)

Reply:
This is still possible, via the translations for the singular form, I've now added a comment to that extent, which should shown on Transifex and so gie the translators a good hint.
IMHO the easier code wins here, and avoids the confusion that as per your idea the singlular would translated but never used.

Reply:
BTW: These singlular/plural things are one reason why we want an en_US 'translation'. Another is some strings from SMuFL which are really British English, but out of our hands to fix.